eis_toolkit/vector_processing/proximity_computation.py

```diff
@@ -0,0 +1,76 @@
+from numbers import Number
+
+import geopandas as gpd
+import numpy as np
+from beartype import beartype
+from beartype.typing import Literal, Optional, Tuple, Union
+from rasterio import profiles
+
+from eis_toolkit.transformations.linear import _min_max_scaling
+from eis_toolkit.vector_processing.distance_computation import distance_computation
+
+
+@beartype
+def proximity_computation(
+    geodataframe: gpd.GeoDataFrame,
+    raster_profile: Union[profiles.Profile, dict],
+    maximum_distance: Number,
+    scaling_method: Literal["linear"],
+    scale_range: Optional[Tuple[Number, Number]] = None,
```

Comment:
I think this shouldn't be optional since it is needed to scale the results. Also could provide a default value here.
```suggestion
    scale_range: Tuple[Number, Number] = (1, 0),
```

Reply:
This was kept optional as I had the log transformations as well for which this scale was not working hence was optional. But will make it as required input with the suggested default value